tiatoolbox/data/pretrained_model.yaml

```diff
@@ -771,3 +771,23 @@ micronet-consep:
             patch_output_shape: [252, 252]
             stride_shape: [150, 150]
             save_resolution: {'units': 'mpp', 'resolution': 0.25}
+
+mapde-crchisto:
+    url: https://tiatoolbox.dcs.warwick.ac.uk/models/detection/mapde-crchisto.pth
+    architecture:
+        class: mapde.MapDe
```

Comment:
You're most probably thinking of adding the `input_resolutions` information here after your have designed the engine. If not, consider adding them based on your training here.

Reply:
I will add them while adding the engine.